src/backend/db/products.js

```diff
@@ -1,258 +1,240 @@
     imgSrc: "https://ik.imagekit.io/mandeshiagro/assets/products/lemon_acfXA-6HJ.png?ik-sdk-version=javascript-1.4.3&updatedAt=1648264697839",
     inStock: true,
     bestDeal: true,
     rating: 3.5,
-    ratingCount: 13,
     isLatest: false,
     inStock: true,
-    userSetQuantity: 1
   },
   {
     _id: uuid(),
     productName: "Pomegranate",
     category: "organicFruits",
     price: "38",
     prevPrice: "40",
+    discount:10,
     imgSrc: "https://ik.imagekit.io/mandeshiagro/assets/products/pomegranate_1n_nCkuNa.jpg?ik-sdk-version=javascript-1.4.3&updatedAt=1648275804075",
     inStock: true,
     bestDeal: false,
     rating: 2.8,
-    ratingCount: 12,
     isLatest: false,
-    userSetQuantity: 1
   },
   {
     _id: uuid(),
     productName: "Chillies",
     category: "organicSpices",
     price: "35",
     prevPrice: "50",
+    discount:12,
     imgSrc: "https://ik.imagekit.io/mandeshiagro/assets/products/chilli_S1Q0D_hFQ.png?ik-sdk-version=javascript-1.4.3&updatedAt=1648264660366",
     inStock: true,
     bestDeal: true,
     rating: 2.5,
-    ratingCount: 11,
     isLatest: false,
-    userSetQuantity: 1
   },
   {
     _id: uuid(),
     productName: "Garlic",
     category: "organicSpices",
     price: "25",
     prevPrice: "35",
+    discount:10,
     imgSrc: "https://ik.imagekit.io/mandeshiagro/assets/products/garlic_gjPzBcbNUg.jpg?ik-sdk-version=javascript-1.4.3&updatedAt=1648277953201",
     inStock: false,
     bestDeal: false,
     rating: 4.1,
-    ratingCount: 14,
     isLatest: true,
-    userSetQuantity: 1
   },
   {
     _id: uuid(),
     productName: "Ginger",
     category: "organicSpices",
     price: "50",
     prevPrice: "55",
+    discount:20,
     imgSrc: "https://ik.imagekit.io/mandeshiagro/assets/products/ginger_c9hlyAsvo.jpg?ik-sdk-version=javascript-1.4.3&updatedAt=1648277953696",
     inStock: true,
     bestDeal: false,
     rating: 4.8,
-    ratingCount: 14,
     isLatest: true,
-    userSetQuantity: 1
   },
   {
     _id: uuid(),
     productName: "Beetroot",
     category: "organicVegetables",
     price: "7",
     prevPrice: "10",
+    discount:2,
     imgSrc: "https://ik.imagekit.io/mandeshiagro/assets/products/beetroot_PBkYRIvvc.png?ik-sdk-version=javascript-1.4.3&updatedAt=1648264706542",
     inStock: false,
     bestDeal: false,
     rating: 1.5,
-    ratingCount: 6,
     isLatest: false,
-    userSetQuantity: 1
   },
   {
     _id: uuid(),
     productName: "Cauli Flower",
     category: "organicVegetables",
     price: "15",
     prevPrice: "20",
+    discount:5,
     imgSrc: "https://ik.imagekit.io/mandeshiagro/assets/products/cauli-flower_Oh_HAtN1x.png?ik-sdk-version=javascript-1.4.3&updatedAt=1648264702792",
     inStock: true,
     bestDeal: false,
     rating: 5,
-    ratingCount: 8,
     isLatest: false,
-    userSetQuantity: 1
   },
   {
     _id: uuid(),
     productName: "Brinjal",
     category: "organicVegetables",
     price: "100",
     prevPrice: "120",
+    discount:20,
```

Comment:
Just a small suggestion , you can always calculate discount on the go if needed because you have both final and previous price in your database, there is no need to store discount also in the database.

Reply:
> liked how you have used commits to commit various features. still, the pull requests having so many commits are tough to review. Please keep that in mind for future pr's .

since I have branched out of wishlist it is showing the commits of wishlist as well